### PR TITLE
Fixes the AI not having access to APC's

### PR DESCRIPTION
**Pull request by lbnesquik**
## About The Pull Request
How did no one fix this before

## Why It's Good For The Game
Just intended functionality
## Changelog
:cl:
fix: Makes it so the AI can access the APC menu.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/silicon/ai/ai.dm [hunk 1]
```diff
@@ -11,6 +11,7 @@
 	sight = SEE_TURFS | SEE_MOBS | SEE_OBJS
 	hud_type = /datum/hud/ai
 	buckle_flags = NONE
+	has_unlimited_silicon_privilege = 1
```
Comment (TiviPlus) [inline code/modules/mob/living/silicon/ai/ai.dm:14]: This should be TRUE

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/silicon/ai/ai.dm (hunk 1 above)
Comment (lbnesquik) [reply code/modules/mob/living/silicon/ai/ai.dm:14]: TRUE

File: code/modules/power/apc.dm [hunk 2]
```diff
@@ -750,7 +750,7 @@
 
 /obj/machinery/power/apc/ui_act(action, list/params)
 	. = ..()
-	if(. || !can_use(usr, TRUE) || locked)
+	if(. || !can_use(usr, TRUE) || locked & (!usr.has_unlimited_silicon_privilege))
```
Comment (TiviPlus) [inline code/modules/power/apc.dm:753]:
```suggestion
	if(. || !can_use(usr, TRUE) || (locked && !usr.has_unlimited_silicon_privilege))
```

Comment (lbnesquik) [reply code/modules/power/apc.dm:753]: Why is this better ?

Comment (BraveMole) [reply code/modules/power/apc.dm]: & is a binary operation, it's boolean math.
&& is logical operation

& evaluates both sides of the operation.
&& evaluates the left side of the operation, if it's true, it continues and evaluates the right side.

**Review by SplinterGP — APPROVED** [review]

**Pull request merged**